project/Dependencies.scala

```diff
@@ -135,13 +135,13 @@ object Dependencies {
       // reactive streams tck
       val reactiveStreamsTck = "org.reactivestreams" % "reactive-streams-tck" % "1.0.2" % "test" // CC0
 
-      val protobufRuntime =  "com.google.protobuf" % "protobuf-java" % "3.9.0" % "test"
+      val protobufRuntime = "com.google.protobuf" % "protobuf-java" % "3.9.0" % "test"
     }
 
     object Provided {
       // TODO remove from "test" config
       // If changed, update akka-docs/build.sbt as well
-      val sigarLoader = "io.kamon" % "sigar-loader" % "1.6.6-rev002" % "optional;provided;test" // ApacheV2
+      val sigarLoader = "io.kamon" % "sigar-loader" % "1.6.6" % "optional;provided;test" // ApacheV2
```

Comment:
https://search.maven.org/search?q=g:io.kamon%20AND%20a:sigar-loader&core=gav suggests 1.6.6-rev002 is newer than 1.6.6 (but there's an even-newer 1.6.6-rev003).

Looks like comment above this line is outdated?

Reply:
Interesting that a lower version number shows as newer https://search.maven.org/search?q=g:io.kamon%20AND%20a:sigar-loader&core=gav so the update plugin was wrong. I'm closing.